segment_executiontime.go

```diff
@@ -35,3 +48,115 @@ func (t *executiontime) init(props *properties, env environmentInfo) {
 	t.props = props
 	t.env = env
 }
+
+func (t *executiontime) formatDuration(ms int64, style DurationStyle) string {
+	switch style {
+	case AustinStyle:
```

Comment:
I would remove the `Style` postfix, the type implies it's a style.

Reply:
done